src/index.js

```diff
@@ -80,6 +81,7 @@ export default class Spinner extends React.Component {
 
   static defaultProps = {
     visible: false,
+    cancelable: true,
```

Comment:
LGTM, probably the only thing I would change it is from `true` to `false`in the default value of the prop. To be disable by default.